resources/js/utils/helpers.ts

```diff
@@ -0,0 +1,11 @@
+export const config = (key: string) => window?.cfg?.[key];
+
+export const asset = (uri: string) => `${config('assetsUrl')}${uri}`;
+
+export const clipboard = (text: string) => {
+  navigator.clipboard.writeText(text).then(() => {
+    if (window.showStatusSuccess) {
+      window.showStatusSuccess('Copied!');
+    }
+  });
```

Comment:
This previously was using the deprecated `document.execCommand` technique for feeding content into the clipboard. I have migrated this to use the [navigator.clipboard API](https://caniuse.com/mdn-api_navigator_clipboard).